Rules.md

```diff
@@ -203,15 +203,16 @@ A battle has one or more rounds. Every round has following parts:
 1. **Attacker decides to retreat or has no healthy units left:** All units move back to the last
    touched unoccupied, sea or owned region.
 1. **Defender decides to retreat or has no healthy units left:** All units (including wounded units)
-   move to an adjacent land region that is unoccupied or owned by the defender.
+   move to a single adjacent land region that is unoccupied or owned by the defender. Exception:
+   units can't retreat to the region where the attacker's move token was resolved.
```

Comment:
>> "units can't retreat to the region where the attacker came from"
>
> Since "came from" isn't defined rigorously, this could mean Ukraine and/or Russia. --> Ambiguous

I don't see how this is not ambiguous? I'm honestly not sure whether I'd be able to retreat to an empty Russia or not.